css/style.css

```diff
@@ -1,6 +1,180 @@
+
 html,
 body {
-  font-family: montserrat-font, sans-serif;
+  font-family: 'montserrat-font', sans-serif;
   font-size: 15px;
   font-weight: normal;
+  color: #000000;
+}
+
+a {
+  text-decoration: none;
+}
+
+.btn {
+  padding: 10px 45px;
+  font-size: 13px;
+  line-height: 16px;
+  font-weight: bold;
+  text-transform: uppercase;
+  background-color: #ffffff;
+  border: 2px solid #E51937;
+  cursor: pointer;
+  transition: 1s;
+}
+
+.btn:hover {
+  transform: scale(1.1);
+}
+
+.header {
+  margin-bottom: 45px;
+}
+
+.container {
+  width: 1440px;
+}
+
+.container-header {
+  display: flex;
+  align-items: center;
+  padding-top: 17px;
+  padding-left: 31px;
+}
+
+.menu {
+  margin-left: auto;
+  margin-right: 68px;
+  margin-bottom: 5px;
+}
+
+.menu__link {
+  position: relative;
+  margin-right: 30px;
+  line-height: 18px;
+  color: #000000;
+}
+
+.menu__link:last-child {
+  margin-right: 0px;
+}
+
+.menu__link:nth-child(5){
+  margin-right: 230px;
+}
+
+.menu__link:nth-child(1) {
+  color: #E51937;
+}
+
+.menu__link:nth-child(1)::before {
+  position: absolute;
+  top: 25px;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  content: "";
+  background-image: url(/img/link-decoration.svg);
+  background-repeat: no-repeat;
+}
+
+.main-section {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-image: url(/img/Tesla-Wordmark.png);
+  background-repeat: no-repeat;
+}
+
+.main-title {
+  margin-top: 0;
+  padding: 0;
+  margin-bottom: 0px;
+  font-size: 90px;
+}
+
+.discription-title {
+  margin: 0;
+  font-weight: medium;
+  line-height: 18px;
+}
+
+.main-section__img-car {
+  margin-top: 25px;
+}
+
+.special-list {
+  display: flex;
+  list-style-type: none;
+  margin-bottom: 25px;
+}
+
+.special-list__item:nth-child(1)  {
+  padding-left: 60px;
+  margin-right: 60px;
+  background-image: url(/img/Speed.png);
+  background-repeat: no-repeat;
+  background-position: left;
+}
+
+.special-list__item:nth-child(2)  {
+  padding-left: 50px;
+  margin-right: 100px;
+  background-image: url(/img/Road\ icon.png);
+  background-repeat: no-repeat;
+  background-position: left;
+}
+
+.special-list__item:nth-child(3)  {
+  padding-left: 30px;
+  margin-right: 40px;
+  background-image: url(/img/steering-wheel.png);
+  background-repeat: no-repeat;
+  background-position: left;
+}
+
+.special-list__item:nth-child(4) {
+  margin-right: 22px;
+  padding-top: 10px;
+}
+
+.special-list-text {
+  margin: 0;
+  font-size: 35px;
+  line-height: 43px;
+}
+
+.special-list-description {
+  font-size: 12px;
+  line-height: 15px;
+  font-weight: lighter;
+}
+
+.special-list-text__description {
+  font-size: 20px;
+  font-weight: medium;
+  line-height: 24px;
+}
+
+.test-drive-text {
+  padding-top: 10px;
+  font-size: 13px;
+  font-weight: bold;
+  line-height: 16px;
+  text-transform: uppercase;
+}
+
+.order-button-red {
+  color: #E51937;
+}
+
+.arrow-downd {
+  transition: 1s;
 }
+
+.arrow-downd:hover {
+  transform: scale(1.2) ;
+}
```

Comment:
Codacy found an issue: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/GurevAleksey_itogovaya/pullRequest?prid=6093420)